package/piwebcam/multi-gadget.sh

```diff
@@ -1,58 +1,74 @@
 #!/bin/sh
-mkdir /sys/kernel/config/usb_gadget/pi4
-
-echo 0x1d6b > /sys/kernel/config/usb_gadget/pi4/idVendor
-echo 0x0104 > /sys/kernel/config/usb_gadget/pi4/idProduct
-echo 0x0100 > /sys/kernel/config/usb_gadget/pi4/bcdDevice
-echo 0x0200 > /sys/kernel/config/usb_gadget/pi4/bcdUSB
-
-echo 0xEF > /sys/kernel/config/usb_gadget/pi4/bDeviceClass
-echo 0x02 > /sys/kernel/config/usb_gadget/pi4/bDeviceSubClass
-echo 0x01 > /sys/kernel/config/usb_gadget/pi4/bDeviceProtocol
-
-mkdir /sys/kernel/config/usb_gadget/pi4/strings/0x409
-echo 100000000d2386db > /sys/kernel/config/usb_gadget/pi4/strings/0x409/serialnumber
-echo "Show-me-webcam Project" > /sys/kernel/config/usb_gadget/pi4/strings/0x409/manufacturer
-echo "Show-me-webcam Pi Webcam" > /sys/kernel/config/usb_gadget/pi4/strings/0x409/product
-mkdir /sys/kernel/config/usb_gadget/pi4/configs/c.2
-mkdir /sys/kernel/config/usb_gadget/pi4/configs/c.2/strings/0x409
-echo 500 > /sys/kernel/config/usb_gadget/pi4/configs/c.2/MaxPower
-echo "Show-me-webcam Pi Webcam" > /sys/kernel/config/usb_gadget/pi4/configs/c.2/strings/0x409/configuration
-
-mkdir /sys/kernel/config/usb_gadget/pi4/functions/uvc.usb0
-mkdir /sys/kernel/config/usb_gadget/pi4/functions/acm.usb0
-mkdir -p /sys/kernel/config/usb_gadget/pi4/functions/uvc.usb0/control/header/h
-ln -s /sys/kernel/config/usb_gadget/pi4/functions/uvc.usb0/control/header/h /sys/kernel/config/usb_gadget/pi4/functions/uvc.usb0/control/class/fs
-
-mkdir -p /sys/kernel/config/usb_gadget/pi4/functions/uvc.usb0/streaming/mjpeg/m/1080p
-cat <<EOF > /sys/kernel/config/usb_gadget/pi4/functions/uvc.usb0/streaming/mjpeg/m/1080p/dwFrameInterval
+
+CONFIG=/sys/kernel/config/usb_gadget/piwebcam
+mkdir -p $CONFIG
+cd $CONFIG
```

Comment:
Those should be quoted and the whole script should be checked against [shellcheck](https://www.shellcheck.net/).

Reply:
I've updated the shell script. On a second thought I think this should be part of @peterbay's uvc-widget repo, as we don't configure anything related to showmewebcam there.